Dockerfile

```diff
@@ -7,6 +7,8 @@ ENV release=v2.0.11 package=zen-2.0.11-amd64.deb
 RUN apt-get update \
     && DEBIAN_FRONTEND=noninteractive apt-get -y --no-install-recommends install apt-utils \
     && DEBIAN_FRONTEND=noninteractive apt-get -y --no-install-recommends install ca-certificates curl wget libgomp1 \
+    && curl -Lo /usr/local/share/ca-certificates/lets-encrypt-x3-cross-signed.crt https://letsencrypt.org/certs/lets-encrypt-x3-cross-signed.pem.txt \
```

Comment:
Thanks for the PR!
Please add hash verification of the downloaded `lets-encrypt-x3-cross-signed.crt` file, e.g. add 
```
&& echo "e446c5e9dbef9d09ac9f7027c034602492437a05ff6c40011d7235fca639c79a  /usr/local/share/ca-certificates/lets-encrypt-x3-cross-signed.crt" | sha256sum -c - \
```